internal/backend/runtime/omni/pkg/check/etcd.go

```diff
@@ -93,12 +93,28 @@ func Etcd(ctx context.Context, r controller.Reader, clusterName string) error {
 		}
 	}
 
-	if (len(members)-1)%2 != 0 {
+	quorum := len(members)/2 + 1
```

Comment:
I think we should calculate the value which makes sense: `loss_we_can_tolerate = len(members) - quorum`,
`optimal_members = quorum*2-1`.

if `loss_we_can_tolerate == 0` => ??? show a warning that we can't tolerate any single member loss

if `len(members) == optimal_members` => OK

if `optimal_members < len(members)` => you can tolerate loss of %d members, but you can improve by adding X members to tolerate loss of %d members


I'm not a content writer person, but the math should be correct imho

Reply:
> if loss_we_can_tolerate == 0 => ??? show a warning that we can't tolerate any single member loss

But only if the count of members is 2.

I don't see much difference of what I have and what you suggest.

On wording I guess we can ask @steverfrancis :)